src/pal/src/synchmgr/wait.cpp

```diff
@@ -75,6 +75,35 @@ WaitForSingleObject(IN HANDLE hHandle,
 
 /*++
 Function:
+  WaitForSingleObjectPrioritized
+
+Similar to WaitForSingleObject, except uses a LIFO release policy for waiting threads by prioritizing new waiters (registering
+them at the beginning of the wait queue rather than at the end).
+--*/
+DWORD
+PALAPI
+PAL_WaitForSingleObjectPrioritized(IN HANDLE hHandle,
```

Comment:
>PAL_ [](start = 0, length = 4)

why using PAL_ in the name here? I am not seeing aw are doing that in other places

Reply:
The PAL tries to mimic the Windows API. For functions that are not in that category I believe the convention is to prefix with PAL_.

Reply:
There are a bunch of functions that follow this convention, for example https://github.com/dotnet/coreclr/blob/master/src/pal/inc/pal.h#L5393